lib/collection/src/operations/conversions.rs

```diff
@@ -585,6 +585,7 @@ impl<'a> From<CollectionSearchRequest<'a>> for api::grpc::qdrant::SearchPoints {
                 DEFAULT_VECTOR_NAME => None,
                 vector_name => Some(vector_name.to_string()),
             },
+            read_consistency: None,
```

Comment:
why is there no param. no forward in this conversion?

Reply:
It's used for `RemoteShard` only, and we don't propagate read consistency over internal API (or everything explode). There are safeguards for this in the internal API service definition also, but kinda makes sense to set `None` here as well.